src/main/java/com/manardenza/utils/DatabaseSeeder.java

```diff
@@ -1,6 +1,66 @@
 package com.manardenza.utils;
 
-public class DatabaseSeeder {
+import com.manardenza.dao.HotelDaoImpl;
+import com.manardenza.dao.UserDaoImpl;
+import com.manardenza.entity.Hotel;
+import com.manardenza.entity.Room;
+import com.manardenza.entity.User;
 
-    // Class to fill DAO implementations
+import java.util.ArrayList;
+import java.util.List;
+
+public final class DatabaseSeeder {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1118)